### ISSUE TITLE
Product name change of plugin: swordserver/swordServer

### ISSUE THREAD

**Issue by ctgraham** [pkp/pkp-lib#7200]
**Describe the bug**
In February the product name for the plugin "swordserver" was changed to "swordServer".  The packages in the [Plugin Gallery's `plugins.xml`](https://github.com/pkp/plugin-gallery/blob/f0a1aca7795cf6b0edf3bcc67c5a03f20c047b7d/plugins.xml#L4693-L4725) reference [0.0.0.1](https://github.com/pkp/swordServer/blob/48a7f3115247de2b967e29f8dfce8494f398813e/version.xml#L14) and [1.0.0.0](https://github.com/pkp/swordServer/blob/61b405fe067e443bd4ba483a2b3c3e865528b2fc/version.xml#L14), which both specify the name of "swordserver".  Prerelease packages of [1.0.1.0](https://github.com/pkp/swordServer/blob/ddf61b8574f16e91c5621bf90e735533c0609c77/version.xml#L14) and [1.1.0.0](https://github.com/pkp/swordServer/blob/30980e7288cfca7e44a921621af5911507cc2e02/version.xml#L14) both specify the camelCase, but are not in the Plugin Gallery (yet).

What was the motivation for changing the name, @asmecher, and what is the distinction of the pre-release packaging from February?

My instinct would be to:
* Rename the product to "swordserver" in `plugins.xml` for the existing releases.
* Either:
  * Create a new product release for "swordServer" for the new releases /or/
  * Update the `version.xml` back to "swordserver" and repackage the pre-releases.

**What application are you using?**
 OJS 3.2.1-4

**Comment by asmecher** [comment]
The repo has always been called "swordServer" (https://github.com/pkp/swordServer) and the plugin build script packages the `.tar.gz` accordingly. We tend to use lowerCamelCase for repo names. The `version.xml` part of this was the only piece of this that was out of sync.

I don't believe the plugin was installable using the plugin gallery or "upload a plugin" tools while this mismatch was in place, as the .tar.gz needs to contain a directory named for the repo and matching the product in `version.xml` -- though it would've been possible to install it manually (e.g. by unpacking from the .tar.gz manually), I suppose. Is that what happened in your case?

**Comment by ctgraham** [comment]
I tried to deploy the plugin from the CLI based on the Plugin Gallery's `plugins.xml` and failed.  Is the resolution to promote the pre-release entries as releases and replace the plugin gallery entries with those?

**Comment by asmecher** [comment]
Yes -- but I was waiting on some third-party test results before doing that. See https://forum.pkp.sfu.ca/t/sword-setup-for-deposits-from-ops-to-ojs/65485/9. If you're able to give these a try, I would be very happy to promote them in the plugin gallery!
